apiary.apib

```diff
@@ -412,6 +520,151 @@ A Shop resource.
             }
         ]
 
+### Create a Shop [POST]
+
+Creates a new shop. The user must be an account administrator
+
++ Request (application/json)
+
+    + Header
+
+            Accept: application/json
+            Authorization: eyJ0eXAiOiJKV1QiLCJhbiAoOiJIUzI1NiJ9.eyJvX2F1dGhfYXBwbGljYXRpb25faWQiOjF9.h7ZNPpbd_Bu8yL27fmCAI_wgyZJI44b5eRRxwWUcwaQ
+    + Body
+
+            {
+                "name": "My Shop",
+                "stablishments": [
+                        {
+                                "number": "12345667",
```

Comment:
this means we are going to both create shop and cascaded down the stablishments and the credentials? so if any of this has an issue, what happens?

Was it in consideration to just create a Shop in this endpoint, and stablishment's endpoint just create stablishment and in that example, fail if any shop_id OR credential_id, doesn't exist? I think i'm coming short on some cases by doing isolated creations, but think it;s worth asking.